packages/admin-ui/src/utils.ts

```diff
@@ -1,6 +1,14 @@
 import { clsx, type ClassValue } from "clsx";
+import { generateId as baseGenerateId } from "@webiny/utils/generateId";
 import { twMerge } from "tailwind-merge";
 
 export function cn(...inputs: ClassValue[]) {
     return twMerge(clsx(inputs));
 }
+
+export const generateId = (initialId?: string): string => {
+    if (initialId !== undefined && initialId !== null) {
+        return String(initialId);
+    }
+    return "wby-" + baseGenerateId(4);
+};
```

Comment:
Couldn't this just be?

```suggestion
export const generateId = (initialId?: string) => {
    if (initialId) {
        return initialId;
    }
    return "wby-" + baseGenerateId(4);
};
```

The other thing is.... I'd maybe just do this `if (initialId...` logic in the Switch component and not bring this extra logic into a simple `generateId` function. Kinda makes it easer to grasp `generateId`. It generates a random ID and that's basically it.

Maybe it's just subjective, so if you'd still leave this as is, let's leave it.